stack.yaml

```diff
@@ -6,22 +6,22 @@ packages:
 - servant-auth-docs
 - servant-auth-swagger
 extra-deps:
-- aeson-0.11.2.1
+- aeson-1.2.2.0
 - bytestring-builder-0.10.8.1.0
 - call-stack-0.1.0
 - fast-logger-2.4.7
-- hspec-2.3.2
-- hspec-core-2.3.2
-- hspec-discover-2.3.2
+- hspec-2.4.4
+- hspec-core-2.4.4
+- hspec-discover-2.4.4
 - hspec-expectations-0.8.2
-- http-api-data-0.3.3
-- jose-0.5.0.0
+- http-api-data-0.3.7.1
+- jose-0.6.0.3
 - mmorph-1.0.9
-- servant-0.9.1.1
-- servant-client-0.9.1.1
-- servant-docs-0.9.1.1
-- servant-server-0.9.1
+- servant-0.11
+- servant-client-0.11
+- servant-docs-0.11
+- servant-server-0.11
 - unix-time-0.3.7
 - wai-extra-3.0.19
 - warp-3.2.9
-resolver: lts-6.7
```

Comment:
We do need CI (and stack files) for all GHCs from 7.10 till 8.2. I think it makes sense for the default `stack.yaml` to be on the latest released GHC (8.0), but we then need `stack-ghc-7.10.3.yaml` (which would essentially be this file before the changes.